resources/keb/values.yaml

```diff
@@ -8,7 +8,7 @@ global:
       path: europe-docker.pkg.dev/kyma-project/prod
     schema_migrator:
       dir: "control-plane"
-      version: "v20231211-c10b7b03"
+      version: "v20230120-865831c1" # do not update along with the other images
```

Comment:
Why such an old version (January!)

Reply:
This is just the newest version used in Control Plane.